### PR TITLE
stars were ordered

### PR DESCRIPTION
**Pull request by Vova-Stoliar**
- [DEMO LINK](https://Vova-Stoliar.github.io/layout_stars/)
- [TEST REPORT LINK](https://Vova-Stoliar.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,11 +3,59 @@
...
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars--0">
```
Comment (herkoss) [inline src/index.html:12]:
```suggestion
    <div class="stars stars--0">
```

Comment (herkoss) [reply src/index.html:12]: Add common class stars to all parent divs

File: src/index.html [hunk 2]
```diff
@@ -3,11 +3,59 @@
...
+      <span class="stars-star"></span>
```
Comment (herkoss) [inline src/index.html:13]:
```suggestion
      <span class="stars__star"></span>
```

Comment (herkoss) [reply src/index.html:13]: Fix here an in the whole file please

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+[class^="stars--"] {
+  display: flex;
+}
```
Comment (herkoss) [inline src/style.css:7]: Don't use selectors like that. Use classes instead

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  background-image: url(./images/star.svg);
+}
+
+.stars--4 > :nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 > :nth-child(n+6) {
+  background-image: url(./images/star.svg);
+}
```
Comment (herkoss) [inline src/style.css:40]: Group these selectors with a comma. Don't repeat yourself

**Review by herkoss — CHANGES_REQUESTED** [review]
Well done!👍👏 
But there are some little mistakes)

**Review by varseniuk — APPROVED** [review]
Good job!

**Pull request closed without merging**